source/net/ServerSocket.ooc

```diff
@@ -127,7 +130,7 @@ ServerSocket: class extends Socket {
 				shutdown(conn sock descriptor, SHUT_RDWR)
 			}
 			conn close()
-			if ((conn && ret) as Bool)
+			if (ret as Bool)
```

Comment:
It was complaining that `conn` is dereferenced even after it is checked for `null`.